### DISCUSSION TITLE
Flex SDK → flex:RateLimiter (WARN):  rate limiter hit. name="flex-ui-interaction-v1"

### DISCUSSION THREAD

**Discussion by darrennix** [twilio-professional-services/flex-project-template#61] [Q&A] [answered]
We've been using this repo as our baseline for rolling out Flex, started about a month ago using the v2 plugin.  In dev I noticed that after just a call or two my console would reach 1MB of events and the Flex SDK would rate limit.  

Thought it was just a dev config somewhere that I couldn't find how to turn off, but it's affecting all our users on Prod since going live.

Example console log after 1 call:
[flex.twilio.com-1665600136399.log](https://github.com/twilio-professional-services/flex-project-template/files/9767863/flex.twilio.com-1665600136399.log)

Any ideas where this debug flag is set and how to turn it off?

Our build is nearly a 1:1 mirror of this starter project template -- we have not added any special logging, changed flags, etc.

**Comment by dremin** [comment]
We've been seeing these logs on Flex UI 2.0.0-beta.1 even without any plugins deployed, so I don't believe this is a direct result of the plugin, and is expected behavior for 2.0.0-beta.1.

**Comment by jared-hunter** [comment]
Very pleased to hear you have been leveraging the template.

As @dremin mentioned, this is unfortunately a flex 2.0 thing - you should be able to update this in the [flex configuration](https://www.twilio.com/docs/flex/developer/plugins/modifying-configuration-for-flextwiliocom) but it is currently ignored.

you can see this setting in action in your `public/appConfig.js` file but that configuration is only applied when running locally

I would advise creating a support ticket in the console to get some weight behind it, i'll do what i can to draw some attention to it internally.

**Comment by darrennix** [comment]
I've got ticket # 9754071 open with support on this issue.

**Comment by jared-hunter** [comment, marked as answer]
After doing some digging around in jira it looks like the fix is already waiting to be released with the next 2.0 release.

as an aside, i think this might be a bit of a red herring for your root issue - i dont think the log level should have anything to do with the rate limit, thats to do with events that are being posted as far as i can tell.  Perhaps you have added some code that's triggering some odd behavior.

I'd recommend doing some testing locally to reproduce and dig deeper, you can lower loglevels using appConfig locally to confirm
